migen/build/platforms/arty_s7.py

```diff
@@ -0,0 +1,116 @@
+# This file is Copyright (c) 2018 William D. Jones <thor0505@comcast.net>
+# License: BSD
+
+from migen.build.generic_platform import *
+from migen.build.xilinx import XilinxPlatform, XC3SProg, VivadoProgrammer
+
+_io = [
+    ("user_led", 0, Pins("E18"), IOStandard("LVCMOS33")),
+    ("user_led", 1, Pins("F13"), IOStandard("LVCMOS33")),
+    ("user_led", 2, Pins("E13"), IOStandard("LVCMOS33")),
+    ("user_led", 3, Pins("H15"), IOStandard("LVCMOS33")),
+
+    ("rgb_leds", 0,
```

Comment:
Shouldn't this be `rgb_led`? And the grouping seems transposed.

Reply:
Litex [opted for](https://github.com/enjoy-digital/litex/search?utf8=✓&q=rgb_leds&type=) `rgb_leds` for the Artix 7 Arty, so I did the same for consistency with what already existed.

I made a sample bitstream, and the grouping is correct (although I understand why it looks transposed. [Digilent's UCF](https://github.com/Digilent/Arty-S7-50-base-uc/blob/master/src/constraints/Arty-S7-50-Master.xdc) doesn't really give any hints to the correct order).